pepys_import/database/default_data/Help Texts.csv

```diff
@@ -18,6 +18,6 @@ FUZZY_SEARCH_SENSOR_TYPE,"Specify the type of sensor."
 FUZZY_SEARCH_NATIONALITY,"Specifythe nationality of this platform"
 KEEP_PLATFORM_AS_SYNONYM,"Add platform name as a synonym. Warning: This should only be done when platform name is a completely unique identifier for this platform not a name that could be shared across platforms of different nationalities."
 DID_NOT_SELECT_EXISTING,"You did not select an existing value. Please search again."
-PLATFORM_IDENTIFIER,"The identifier of platform, such as a Pennant number for a warship or tail number in an aircraft. Where an identifer isn't available for that platform-type, the platform name should be entered as identifier. Up to 10 characters are allowed."
+PLATFORM_IDENTIFIER,"The identifier of platform, such as a Pennant number for a warship or tail number in an aircraft. Where an identifer isn't available for that platform-type, the platform name should be entered as identifier. Up to 30 characters are allowed."
```

Comment:
well spotted @BarisSari  :-)

Reply:
Thanks. I checked the "10" string in the whole project :)